nozzle/event_handler.go

```diff
@@ -48,8 +44,8 @@ func CreateEventHandler(conf *WavefrontConfig) EventHandler {
 	if len(conf.URL) > 0 && len(conf.Token) > 0 {
 		logger.Printf("Direct connetion to Wavefront: %s", conf.URL)
 		directCfg := &senders.DirectConfiguration{
-			Server:               conf.URL,
-			Token:                conf.Token,
+			Server:               strings.Trim(conf.URL, " "),
+			Token:                strings.Trim(conf.Token, " "),
 			BatchSize:            10000,
 			MaxBufferSize:        50000,
```

Comment:
default to 100,000. Also make this configurable (that was merged in one of the other branches).